src/amo/reducers/currentView.js

```diff
@@ -0,0 +1,38 @@
+/* @flow */
+import { CURRENT_VIEW_SET } from 'core/constants';
+
+
+export type currentViewTypes = {|
+  // TODO: Create an addonType Type that is one of type (ADDON_TYPE_EXTENSION,
+  // ADDON_TYPE_THEME, etc.)
+  addonType: string | null,
+  isExploring: boolean | void,
+  isHomePage: boolean | void,
```

Comment:
Exploring happens when you're searching both extensions and themes but also when you're on the home page; it's basically anytime you are not specifically in extensions or themes. So:

`/` = `isHomePage === true`, `isExploring === true`
`/search?q=foo` = `isHomePage === false`, `isExploring === true`
`/search?q=foo&type=extension` = `isHomePage === false`, `isExploring === false`